<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hyperentanglementQKD</class>
 <widget class="QWidget" name="hyperentanglementQKD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLCDNumber" name="SubSpace1ZCount">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;子空间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="text">
    <string>Z基计数</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z基QBER</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="SubSpace1ZQBER">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="SubSpace1XCount">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="SubSpace1XQBER">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>子空间1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="text">
    <string>X基计数</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X基QBER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X基QBER</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="SubSpace2XCount">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="SubSpace2XQBER">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>子空间2：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="text">
    <string>Z基计数</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z基QBER</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="SubSpace2ZCount">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;子空间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="text">
    <string>X基计数</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="SubSpace2ZQBER">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4维全空间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X基QBER</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="FullSpaceXCount">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="FullSpaceXQBER">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="text">
    <string>Z基计数</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z基QBER</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="FullSpaceZCount">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;子空间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="text">
    <string>X基计数</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="FullSpaceZQBER">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStartQKD">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStopQKD">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRetornQKD">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="QKDWidgetset">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <widget class="QWidget" name="delayset">
    <attribute name="title">
     <string>延时设置</string>
    </attribute>
    <widget class="QLineEdit" name="TDC1delay1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC1delay2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC1delay3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC1delay4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC1delay5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC1delay6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC2delay2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC2delay3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC2delay4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC2delay5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC2delay6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TDC2delay1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>81</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>延时（ns）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>从左往右</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>从上往下1-6通道</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Zperiod">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Z 基周期（ns）</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="qkdsaveset">
    <attribute name="title">
     <string>保存数据</string>
    </attribute>
    <widget class="QPushButton" name="buttonStopRecord">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>75</width>
       <height>56</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>停止记录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStartRecord">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>75</width>
       <height>57</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>开始记录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenDataDir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>数据目录</string>
     </property>
    </widget>
    <widget class="QComboBox" name="textDataDirectory">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>D:/Data</string>
      </property>
     </item>
    </widget>
    <widget class="QToolButton" name="buttonDataDirectory">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>37</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="textDataFileName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件名</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDataFileSuffix">
     <property name="geometry">
      <rect>
       <x>198</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>txt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dat</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
